aws/cloudformation/rdp_bastion.yaml

```diff
@@ -0,0 +1,492 @@
+      ami: ami-0d5142f63c808d143
+    ca-central-1:
+      ami: ami-04609b5b156500b17
+    sa-east-1:
+      ami: ami-0b123273fd25fe833
+    ap-southeast-1:
+      ami: ami-0fcd2e9ac9a168217
+    ap-southeast-2:
+      ami: ami-09fb3a51968a858cd
+    eu-central-1:
+      ami: ami-0087bd2f5c26af5ca
+    us-east-1:
+      ami: ami-005b11f8b84489615
+    us-east-2:
+      ami: ami-08778753ef37aa408
+    us-west-1:
+      ami: ami-0267fc24ee0102728
+    us-west-2:
+      ami: ami-081aaface2871d0d0
+
+Resources:
+  # Customer VPC
+  rCustomerVpc:
+    Type: AWS::EC2::VPC
+    Properties:
+      CidrBlock:
+        Ref: pCustomerVpcCidrBlock
+      EnableDnsSupport: true
+      EnableDnsHostnames: true
+      Tags:
+        - Key: Name
+          Value: !Join ["", [!Ref pTagprefix, "-customer-vpc"]]
+  # Default Network ACLs (see https://access.redhat.com/documentation/en-us/
+  # red_hat_openshift_service_on_aws/4/html-single/
+  # install_rosa_classic_clusters/index#osd-aws-privatelink-required-resources.
+  # adoc_rosa-aws-privatelink-creating-cluster)
+  rInboundNetworkACLRule1:
+    Type: AWS::EC2::NetworkAclEntry
+    Properties:
+      NetworkAclId:
+        Fn::GetAtt:
+          - rCustomerVpc
+          - DefaultNetworkAcl
+      RuleNumber: 50
+      Protocol: 6
+      RuleAction: allow
+      CidrBlock: 0.0.0.0/0
+      PortRange:
+        From: 1024
+        To: 65535
+  rInboundNetworkACLRule2:
+    Type: AWS::EC2::NetworkAclEntry
+    Properties:
+      NetworkAclId:
+        Fn::GetAtt:
+          - rCustomerVpc
+          - DefaultNetworkAcl
+      RuleNumber: 51
+      Protocol: 6
+      RuleAction: allow
+      CidrBlock: 0.0.0.0/0
+      PortRange:
+        From: 80
+        To: 80
+  rInboundNetworkACLRule3:
+    Type: AWS::EC2::NetworkAclEntry
+    Properties:
+      NetworkAclId:
+        Fn::GetAtt:
+          - rCustomerVpc
+          - DefaultNetworkAcl
+      RuleNumber: 52
+      Protocol: 6
+      RuleAction: allow
+      CidrBlock: 0.0.0.0/0
+      PortRange:
+        From: 443
+        To: 443
+  rInboundNetworkACLRule4:
+    Type: AWS::EC2::NetworkAclEntry
+    Properties:
+      NetworkAclId:
+        Fn::GetAtt:
+          - rCustomerVpc
+          - DefaultNetworkAcl
+      RuleNumber: 53
+      Protocol: 6
+      RuleAction: allow
+      CidrBlock: 0.0.0.0/0
+      PortRange:
+        From: 22
+        To: 22
+  rInboundNetworkACLRule5:
+    Type: AWS::EC2::NetworkAclEntry
+    Properties:
+      NetworkAclId:
+        Fn::GetAtt:
+          - rCustomerVpc
+          - DefaultNetworkAcl
+      RuleNumber: 54
+      Protocol: 6
+      RuleAction: allow
+      CidrBlock: 0.0.0.0/0
+      PortRange:
+        From: 3389
+        To: 3389
+  rInboundNetworkACLRule6:
+    Type: AWS::EC2::NetworkAclEntry
+    Properties:
+      NetworkAclId:
+        Fn::GetAtt:
+          - rCustomerVpc
+          - DefaultNetworkAcl
+      RuleNumber: 60
+      Protocol: -1
+      RuleAction: deny
+      CidrBlock: 0.0.0.0/0
```

Comment:
Those default network ACL were for the ROSA cluster. A customer VPC would not have those. Maybe remove all those ACLs and keep the defaults (allow everything), or deny everything (as it was the initial plan: https://drive.google.com/file/d/1PmRi_iKmiVR4pnHxJFJtEs-wEkURNRat/view?usp=sharing) and then add network ACL at the subnet level for just what you need.

Reply:
Thanks, will change the nacl rules accordingly in the next commit.